drone_localization/api/endpoints/localize.py

```
@@ -19,32 +22,58 @@
+        logger.info("Response")
         return LocalizationResponse(
             drone_image=request.drone_image,  # Как на схеме
-            satellite_image=satellite_result.image,  # Как на схеме
-            coordinates={"lat": lat, "lon": lon},  # Как на схеме
+            satellite_image=satellite_b64,  # Как на схеме
+            coordinates=Coordinates(lat=latitude, lon=longitude),  # Точечные координаты (lat, lon)
             confidence=satellite_result.score,  # Как на схеме
         )
     except HTTPException:
         raise
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)